_build_dependencies.sh

```diff
@@ -0,0 +1,2 @@
+# This file has all the information on what versions of libraries are thrown into the code
+export SPACEMANDMM_TAG=suite-1.2
```

Comment:
This will have more added to it (Node version, DM version) when I finish the travis overhaul